e2e/tests/components/infinite-scroll/standard/infinite-scroll.e2e-spec.ts

```diff
@@ -143,13 +143,15 @@ describe('Infinite Scroll Tests', () => {
         expect(await page.getNumberOfEmployees()).toEqual(38);
 
         // change page size to 39
+        await page.clickOnLoadOnScroll();
         await page.clickOnPageSize();
-        await page.getPageSize().sendKeys(Key.DELETE);
-        await page.getPageSize().sendKeys('3'); // 39
+        await page.getPageSize().clear();
+        await page.getPageSize().sendKeys('39'); // 39
+
         expect(await page.getNumberOfEmployees()).toEqual(39);
-        await page.hoverOverLastEmployee();
+        await page.loadMoreButton.click();
```

Comment:
Could you describe the problem which required this change in the test script?

Reply:
In the micro focus site it didn't consistently load the the next page, so it would work one time and not work the next. So it would always remain at '39' rather than load the rest of the employees when using <code>hoverOverLastEmployee</code>.

Reply:
Do you know why this was specific to the Micro Focus theme?